src/test/java/com/waverley/testcases/Browser.java

```diff
@@ -0,0 +1,43 @@
+package com.waverley.testcases;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+import static org.testng.AssertJUnit.assertEquals;
+
+public class Browser {
```

Comment:
try to revise name for you test class, as seems like there's a mess of browser management staff and tests' logic